src/main/scala/BintrayPlugin.scala

```diff
@@ -0,0 +1,194 @@
+package bintray
+
+import bintry.Attr
+import sbt.{ AutoPlugin, Credentials, Global, Path, Resolver, Setting, Task, Tags, ThisBuild }
+import sbt.Def.{ Initialize, setting, task, taskDyn }
+import sbt.Keys._
+import sbt.Path.richFile
+
+object BintrayPlugin extends AutoPlugin {
```

Comment:
I guess now is a good time to ask, what _is_ an `AutoPlugin` compared to a `Plugin`?